internal/dataplane/parser/translators/httproute_atc.go

```diff
@@ -134,6 +134,12 @@ func generateMatcherFromHTTPRouteMatch(match gatewayapi.HTTPRouteMatch) atc.Matc
 		matcher.And(headerMatcher)
 	}
 
+	// REVIEW: Since KIC 3.0.0 only support Kong 3.4.1 and above, could we omit the check of Kong version here?
```

Comment:
I would say so. I think we should only have the initial check to ensure we talk to Kong version >= 3.4.1 (that is being added in https://github.com/Kong/kubernetes-ingress-controller/pull/4766).

Reply:
removed the comment.